components/singleComment.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const SingleComment = ({ comment }) => {
+  return (
+    <div className="single-comment-container">
+      <div className="single-article-line">
+        <p className="article-author">Commenter: {comment.author}</p>
+        <p className="article-created">Created at:{comment.created_at}</p>
```

Comment:
This time format's a bit difficult to read, consider using a lightweight library like Day.js https://day.js.org/ to make it a bit easier for the user to read 🕐